_includes/people.html

```diff
@@ -21,6 +21,10 @@
           {% if user.twitter %}
           <a class="people-card-meta-twitter" href="https://twitter.com/{{ user.twitter }}"><i class="fab fa-twitter"></i> @{{ user.twitter }}</a><br>
           {% endif %}
+          {% if user.mastodon %}
+          <a class="people-card-meta-mastodon" href="https://mastodon.social/@{{ user.mastodon }}"><i class="fab fa-mastodon"></i> 
```

Comment:
You probably also need to give the mastodon server. Not everyone is on mastodon.social. I am on piaille.fr for example: https://piaille.fr/@bebatut

Reply:
Ah, thanks a lot! I saw some misleading info about `mastadon.social` being the default. I'll revert now!